ui/shared/components/table/LoadReportTable.jsx

```diff
@@ -54,7 +54,7 @@ const ReportTable = React.memo((
       striped
       collapsing
       horizontalScroll
-      downloadFileName={`${viewAllPages.find(({ path }) => path === projectGuid)?.downloadName || (data?.length && data[0][PROJECT_ID_FIELD].replace(/ /g, '_'))}_${new Date().toISOString().slice(0, 10)}_${fileName}`}
+      downloadFileName={`${viewAllPages.find(({ path }) => path === projectGuid)?.downloadName || (data?.length && data[0][PROJECT_ID_FIELD] && data[0][PROJECT_ID_FIELD].replace(/ /g, '_'))}_${new Date().toISOString().slice(0, 10)}_${fileName}`}
```

Comment:
I was getting something like 'can't use .replace on undefined' during local testing, I think because I wasn't using internal projects. We want variant metadata reports to be available for non-internal projects?

Reply:
> We want variant metadata reports to be available for non-internal projects?

No